cfme/test_framework/sprout/client.py

```diff
@@ -88,9 +88,9 @@ def from_config(cls, **kwargs):
         user_key = kwargs.pop('sprout_user_key') if 'sprout_user_key' in kwargs else None
         # First choose env var creds, then look in kwargs for a sprout_user_key to lookup
         user = (os.environ.get("SPROUT_USER") or
-                credentials.get(user_key, {}).get("username") if user_key else None)
+                (credentials.get(user_key, {}).get("username") if user_key else None))
```

Comment:
**REQUIRED** lets use a extra variable storing `credential = credentials.get(user_key, {}) if user_key else {}` to break down repeat statement complexity